nncf/experimental/onnx/graph/metatypes/onnx_metatypes.py

```diff
@@ -29,25 +28,51 @@ def get_all_aliases(cls) -> List[str]:
         return cls.op_names
 
 
+class OpWeightDef:
+    """
+    Contains information about the weight of operation.
+    """
+
+    def __init__(self, weight_port_id: Optional[int], channel_axis: int, bias_port_id: Optional[int]):
+        """
+        Initializes a definition of the weight.
+
+        :param weight_port_id: Input port of the node's weight.
+        :param channel_axis: Axis for per-channel quantization.
+        :param bias_port_id: Input port of the node's bias.
+        """
+        self.weight_port_id = weight_port_id
+        self.channel_axis = channel_axis
+        self.bias_port_id = bias_port_id
+
+
+class ONNXOpWithWeightsMetatype(ONNXOpMetatype):
+    weight_definition = None  # type: OpWeightDef
+
+
 @ONNX_OPERATION_METATYPES.register()
-class ONNXConvolutionMetatype(ONNXOpMetatype):
+class ONNXConvolutionMetatype(ONNXOpWithWeightsMetatype):
     name = 'ConvOp'
     op_names = ['Conv']
     hw_config_names = [HWConfigOpName.CONVOLUTION]
+    weight_definitions = OpWeightDef(weight_port_id=1, channel_axis=0, bias_port_id=2)
```

Comment:
Are you sure that the channel axis in the Convolution is 0? Is it the same `channel_axis` that uses in the `FastBiasCorrection`?

Reply:
It is not clear from the name of the variable and not clear even from the parameter description.

Reply:
Done